source/unix/openssl_platform_init.c

```diff
@@ -140,8 +141,12 @@ bool s_resolve_hmac_102(void *module) {
     /* were symbols bound by static linking? */
     bool has_102_symbols = init_fn && clean_up_fn && update_fn && final_fn && init_ex_fn;
     if (has_102_symbols) {
-        AWS_LOGF_DEBUG(AWS_LS_CAL_LIBCRYPTO_RESOLVE, "found static libcrypto 1.0.2 HMAC symbols");
+        AWS_LOGF_DEBUG(AWS_LS_CAL_LIBCRYPTO_RESOLVE, "found loaded libcrypto 1.0.2 HMAC symbols");
```

Comment:
found loaded symbols sounds kinda weird. maybe found "weak ref" symbols?
and then somewhere before then we should say that we are trying to resolve in process symbols